ozimos/Book-A-Meal
# 156993364 chore server setup

#### What does this PR do?
Sets up nodejs, linting, transpiling and a basic express app in the project

#### Description of Task to be completed?
Set up server side environment in preparation for building out API endpoints

#### How should this be manually tested?
Open a terminal and
* Clone the repository
* Run **npm install**
* Run **npm start**
A confirmation message will appear on the terminal

#### Any background context you want to provide?
Nodejs, ESLint, Babel Transpiling are requirements for the project build

#### What are the relevant pivotal tracker stories? (If applicable)
#156993364

#### Screenshots (If applicable)
None
